src/core/edgedb/dto.repository.ts

```diff
@@ -272,12 +272,17 @@ export const RepoFor = <
     }
   }
 
+  type DefaultRepoOwnKeys = Exclude<
+    keyof DefaultDtoRepository,
+    keyof Repository
+  > &
+    string;
   const DefaultMethods = makeEnum({
     values: entries(
       Object.getOwnPropertyDescriptors(DefaultDtoRepository.prototype),
     ).flatMap(([key]) =>
-      typeof key === 'string'
-        ? (key as keyof DefaultDtoRepository & string)
+      typeof key === 'string' && key !== 'constructor'
+        ? (key as DefaultRepoOwnKeys)
```

Comment:
Oooh, it's overwriting the customized constructor? Wow! Good catch!

Reply:
No no. It was just adding `'constructor'` as an enum member.
```ts
DefaultMethods.has('constructor') // true
```
This wasn't currently hurting anything, just clean up.